libs/gpg45-evaluator/src/main/java/uk/gov/di/ipv/core/library/gpg45/validators/Gpg45DcmawValidator.java

```diff
@@ -13,6 +16,22 @@ public static boolean isSuccessful(CredentialEvidenceItem item) {
         if (item.getFailedCheckDetails() != null) {
             return false;
         }
-        return item.getValidityScore() != 0;
+
+        return item.getValidityScore() != 0
+                && getDcmawVerificationScore(item.getCheckDetails()) > 0
+                && item.getStrengthScore() > 0;
+    }
+
+    private static Integer getDcmawVerificationScore(List<CheckDetail> checkMethods) {
+        var checkMethodWithVerificationScore =
+                checkMethods.stream()
+                        .filter(
+                                checkMethod ->
+                                        checkMethod.getBiometricVerificationProcessLevel() != null)
+                        .findFirst();
+
+        return checkMethodWithVerificationScore.isPresent()
+                ? checkMethodWithVerificationScore.get().getBiometricVerificationProcessLevel()
+                : 0;
```

Comment:
Do you think there's anywhere to commonise this? Perhaps it should be a method on `CredentialEvidenceItem` itself.

I might just override `getVerificationScore` itself so it follows this logic for DCMAW.

Reply:
Not sure it should be in `CredentialEvidenceItem` as I don't think it's a method that should be available if it wasn't from dcmaw. But I agree with replacing `getVerificationScore` instead!

Reply:
You could still include the DCMAW check if it was part of `CredentialEvidenceItem`, e.g.:
```
public Integer getVerificationScore() {
    if (isDcmaw()) {
        // get biometric verification score
    }
    return verificationScore;
}
```
but either way is fine

Reply:
good point!